Public/Src/Utilities/Utilities/BuildXLReaderExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace BuildXL.Utilities
+{
+    /// <summary>
+    /// Extensions for <see cref="BuildXLReader"/>.
+    /// </summary>
+    public static class BuildXLReaderExtensions

```

Comment:
You should add your char read & write methods here

Reply:
See comment in the `FileAccessManifest.cs`. 

---
In reply to: [276827527](https://github.com/Microsoft/BuildXL/pull/159#discussion_r276827527) [](ancestors = 276827527)